db/schema.rb

```diff
@@ -332,6 +339,13 @@
   add_index "slack_channels", ["stage_id"], name: "index_slack_channels_on_stage_id", using: :btree
   add_index "secrets", ["id"], name: "index_secrets_on_id", unique: true, length: {"id"=>191}, using: :btree
 
+  create_table "slack_identifiers", force: :cascade do |t|
+    t.integer  "user_id",    limit: 4
+    t.text     "identifier", limit: 65535, null: false
```

Comment:
nope, have to do that

On Fri, Jul 15, 2016 at 2:09 AM, Zehua Liu <notifications@github.com> wrote:

> In db/schema.rb
> <https://github.com/zendesk/samson/pull/1130#discussion_r70942887>:
>
> > @@ -332,6 +339,13 @@
> >    add_index "slack_channels", ["stage_id"], name: "index_slack_channels_on_stage_id", using: :btree
> >    add_index "secrets", ["id"], name: "index_secrets_on_id", unique: true, length: {"id"=>191}, using: :btree
> >
> > +  create_table "slack_identifiers", force: :cascade do |t|
> > +    t.integer  "user_id",    limit: 4
> > +    t.text     "identifier", limit: 65535, null: false
>
> Should we define indices for these two columns, since we run a number of
> queries on either of them? Or is it magically done by rails?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/f9ac922e76d375774c3b8681d0483f89608f4ed1#r70942887>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ5_ClBVVaoPslwG2fDArO657MzCrks5qV040gaJpZM4JIRHk>
> .
>